tests/utils/test_exceptions.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2024 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import pickle
+from typing import Dict
+
+from llmfoundry.utils.exceptions import (
```

Comment:
Instead of this (cause we will miss new ones), can you iter everything in the exceptions file and find all the exception classes to test?